crm_project_task/views/crm_lead.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Moduon Team S.L.
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0) -->
+<odoo>
+    <record id="inherit_crm_lead_view_form_crm_project_task" model="ir.ui.view">
+        <field name="name">crm.lead.view.form.inherit.crm.project.task</field>
+        <field name="model">crm.lead</field>
+        <field name="inherit_id" ref="crm.crm_lead_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="header" position="inside">
+                <button
+                    string="Create Task"
+                    name="%(wizard_crm_create_task_action)d"
+                    type="action"
+                    class="oe_highlight"
```

Comment:
I don't think this button should be highlighted. It has too much visual weight: 
![imagen](https://github.com/OCA/crm/assets/973709/30063e3d-447b-4909-9e17-a99cda5fea47)

Highlighted buttons are meant to lead the user to the next natural step in a workflow. For a lead, the natural workflow is to send a quotation and win the lead. Creating a task is a divergent workflow, not the natural one; so I don't think it should be highlighted.

Reply:
@rafaelbn is right